applications/GeoMechanicsApplication/tests/test_triangle_10n.gid/test_triangle_10n.mdpa

```diff
@@ -0,0 +1,207 @@
+
+Begin Properties 1
+End Properties
+
+
+Begin Nodes
+  1  0.0000000000 0.0000000000 0.0000000000
+  2  0.1666666667 0.0000000000 0.0000000000
+  3  0.3333333333 0.0000000000 0.0000000000
+  4  0.5000000000 0.0000000000 0.0000000000
+  5  0.6666666667 0.0000000000 0.0000000000
+  6  0.8333333333 0.0000000000 0.0000000000
+  7  1.0000000000 0.0000000000 0.0000000000
+  8  1.0000000000 0.1666666667 0.0000000000
+  9  1.0000000000 0.3333333333 0.0000000000
+  10 1.0000000000 0.5000000000 0.0000000000
+  11 1.0000000000 0.6666666667 0.0000000000
+  12 1.0000000000 0.8333333333 0.0000000000
+  13 1.0000000000 1.0000000000 0.0000000000
+  14 0.8333333333 1.0000000000 0.0000000000
+  15 0.6666666667 1.0000000000 0.0000000000
+  16 0.5000000000 1.0000000000 0.0000000000
+  17 0.3333333333 1.0000000000 0.0000000000
+  18 0.1666666667 1.0000000000 0.0000000000
+  19 0.0000000000 1.0000000000 0.0000000000
+  20 0.0000000000 0.8333333333 0.0000000000
+  21 0.0000000000 0.6666666667 0.0000000000
+  22 0.0000000000 0.5000000000 0.0000000000
+  23 0.0000000000 0.3333333333 0.0000000000
+  24 0.0000000000 0.1666666667 0.0000000000
+  25 0.5000000000 0.5000000000 0.0000000000
+  26 0.1666666667 0.5000000000 0.0000000000
+  27 0.3333333333 0.5000000000 0.0000000000
+  28 0.6666666667 0.5000000000 0.0000000000
+  29 0.8333333333 0.5000000000 0.0000000000
+  30 0.5000000000 0.1666666667 0.0000000000
+  31 0.5000000000 0.3333333333 0.0000000000
+  32 0.5000000000 0.6666666667 0.0000000000
+  33 0.5000000000 0.8333333333 0.0000000000
+  34 0.6666666667 0.1666666667 0.0000000000
+  35 0.8333333333 0.3333333333 0.0000000000
+  36 0.1666666667 0.1666666667 0.0000000000
+  37 0.3333333333 0.3333333333 0.0000000000
+  38 0.6666666667 0.6666666667 0.0000000000
+  39 0.8333333333 0.8333333333 0.0000000000
+  40 0.1666666667 0.6666666667 0.0000000000
+  41 0.3333333333 0.8333333333 0.0000000000
+  42 0.3333333333 0.1666666667 0.0000000000
+  43 0.8333333333 0.1666666667 0.0000000000
+  44 0.1666666667 0.3333333333 0.0000000000
+  45 0.6666666667 0.3333333333 0.0000000000
+  46 0.3333333333 0.6666666667 0.0000000000
+  47 0.8333333333 0.6666666667 0.0000000000
+  48 0.1666666667 0.8333333333 0.0000000000
+  49 0.6666666667 0.8333333333 0.0000000000
+End Nodes
+
+
+Begin Elements SmallStrainUPwDiffOrderElement2D10N
```

Comment:
is it possible to reduce number of elements in this test to reduce the size of mdpa file?

Reply:
Reduced the number of elements from 8 to 4.